centos_ci.sh

```diff
@@ -158,7 +159,7 @@ else
 
 	# setup to run e2e tests
 	cd payload
-	make
+	make release
```

Comment:
Should we be making releases by default? Is there any differences left between release/non-release bundles?